00_Website/assets/css/styles-home-page.css

```diff
@@ -465,8 +445,761 @@ input::placeholder,
   margin-right: 16px;
 }
 
-/* @media all and (min-width: 501px) {
+.project-languages5 .long {
+  display: inline-block;
+}
+
+.project-languages5 .short {
+  display: none;
+}
+
+.information-project5 .long {
+  display: inline-block;
 }
-@media all and (min-width: 1001px) {
 
-} */
+.information-project5 .short {
+  display: none;
+}
+
+@media all and (min-width: 1440px) {
```

Comment:
- [ ] I am glad to see this media query which will handle breakpoints for large screens. However, I think `1440px` is way too high if we want this website to look great on other devices too. In that light, I will suggest you use a value between `992px` and `1100px`.